bundle/definition/schema.go

```diff
@@ -131,10 +131,10 @@ func (s *Schema) ConvertValue(val string) (interface{}, error) {
 		default:
 			return false, errors.Errorf("%q is not a valid boolean", val)
 		}
-	case "object":
+	case "array", "object":
```

Comment:
oops! I see now that I should have added back number to this switch. I'll update my other PR